src/components/Navigation/Navigation.jsx

```diff
@@ -12,14 +13,12 @@ import Dropdown from '../Dropdown/Dropdown';
 import isClient from '../../utilities/is-client';
 
 // Load Styling
-import 'docsearch.js/dist/cdn/docsearch.css';
 import './Navigation.scss';
-import './Search.scss';
+import '@docsearch/css';
```

Comment:
This won't work at the moment, see https://github.com/algolia/docsearch/pull/991